### PR TITLE
feat: support helm plugin default value

### PR DESCRIPTION
**Pull request by steinliber**
Signed-off-by: Meng JiaFeng <jiafeng.meng@merico.dev>

## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/devstream-io/devstream/blob/main/CONTRIBUTING.md) documentation.
- [x] My code has the necessary comments and documentation (if needed).
- [x] I have added relevant tests

## Description
- plugin `argocd` support default value config
- add unit test for helm installer
- all helm options add `mapstructure` field name
- helm validate func not execute default logic

## Related Issues
#514

### CODE REVIEW

File: pkg/util/helm/helm.go [hunk 1]
```diff
@@ -139,3 +139,13 @@ func GetAnnotationName() string {
...
+
+func GetBoolTrueAddress() *bool {
+	trueValue := true
+	return &trueValue
+}
+
+func GetBoolFalseAddress() *bool {
+	falseValue := false
+	return &falseValue
+}
```
Comment (iyear) [inline pkg/util/helm/helm.go:151]: https://github.com/google/go-github/blob/master/github/github.go#L1352-L1366

We can reuse these functions that convert type to pointers by placing them in `pkg/util/types`

File: pkg/util/helm/param.go [hunk 2]
```diff
@@ -9,22 +9,57 @@ type HelmParam struct {
...
+		if defaultField == nil {
+			boolAddress = GetBoolFalseAddress()
+		} else {
+			boolAddress = defaultField
+		}
+	} else {
+		boolAddress = field
+	}
+	return boolAddress
+}
```
Comment (iyear) [inline pkg/util/helm/param.go:65]:
```go
func getBoolValue(field, defaultField *bool) *bool {
	if field != nil {
		return field
	}
	
	if defaultField != nil {
		return defaultField
	}
	
	return types.Bool(false)
}
```

**Review by iyear — APPROVED** [review]
Elegant Impl! Users will have a better idea of what they need to fill in.

**Review by aFlyBird0 — APPROVED** [review]
lgtm

**Review by IronCore864 — APPROVED** [review]

**Pull request merged**